graphiti_core/driver.py

```
@@ -0,0 +1,198 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import logging
+from abc import ABC, abstractmethod
+from typing import Any, Coroutine
+from datetime import datetime
+
+from neo4j import AsyncGraphDatabase, GraphDatabase
```

Ruff (0.11.9)

`neo4j.GraphDatabase` imported but unused

Remove unused import: `neo4j.GraphDatabase`

(F401)